### PR TITLE
Adding test multiple sc with replica 2 and 3 with IO

### PR DESCRIPTION
**Pull request by shyRozen**
Test multiple sc with replica 2 and 3 with 10 pods

### CODE REVIEW

**Review by ebenahar — DISMISSED** [review]
Great PR @shyRozen!

File: tests/manage/storageclass/test_create_2sc_at_once_with_io.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+        replicas = dict()
+        replicas[sc_obj1.name] = 2
+        replicas[sc_obj2.name] = 3
+        sc_obj_list = [sc_obj1, sc_obj2]
+
+        log.info("Creating pvc and pods")
+        pod_obj_list = []
+        for sc_obj in sc_obj_list:
+            for pod_num in range(1, 10):
```
Comment (ebenahar) [inline tests/manage/storageclass/test_create_2sc_at_once_with_io.py:71]: isn't 20 pods running IO too much?

Comment (shyRozen) [reply tests/manage/storageclass/test_create_2sc_at_once_with_io.py:71]: 10?

Comment (ebenahar) [reply tests/manage/storageclass/test_create_2sc_at_once_with_io.py:71]: sounds better. Please test how much load it is doing. You can run with `--log-cluster-utilization` over DC-CP

Comment (shyRozen) [reply tests/manage/storageclass/test_create_2sc_at_once_with_io.py:71]: As default fio is 1m rate cluster util at max was around 60m.

Comment (shyRozen) [reply tests/manage/storageclass/test_create_2sc_at_once_with_io.py:71]: With 20 pods

Comment (shyRozen) [reply tests/manage/storageclass/test_create_2sc_at_once_with_io.py]: Lowered to 10 pods

**Review by ebenahar — APPROVED** [review]

**Review by OdedViner — APPROVED** [review]
LGTM

**Pull request merged**